### PR TITLE
fix spec

### PR DESCRIPTION
**Pull request by herminiotorres**
The square method waits for a tuple with an atom and an integer or a tuple with an atom and a string on the test.
And for the total method waits for a tuple with an atom and an integer.

### CODE REVIEW

**Comment by neenjaw** [comment]
Do you think it might be more descriptive to have it `{:ok, pos_integer()} | {:error, String.t()}`?

**Comment by herminiotorres** [comment]
> Do you think it might be more descriptive to have it `{:ok, pos_integer()} | {:error, String.t()}`?

Good point, I will send this changing, thanks to the point that!

File: exercises/practice/grains/lib/grains.ex [hunk 1]
```diff
@@ -2,14 +2,14 @@ defmodule Grains do
...
-  @spec square(pos_integer) :: pos_integer
+  @spec square(pos_integer) :: {:ok, pos_integer} | {:error, String.t()}
   def square(number) do
   end
 
   @doc """
   Adds square of each number from 1 to 64.
   """
-  @spec total :: pos_integer
+  @spec total :: {atom, pos_integer}
```
Comment (neenjaw) [inline exercises/practice/grains/lib/grains.ex:12]:
```suggestion
  @spec total :: {:ok, pos_integer()}
```

**Review by neenjaw — CHANGES_REQUESTED** [review]

File: exercises/practice/grains/lib/grains.ex (hunk 1 above)
Comment (neenjaw) [reply exercises/practice/grains/lib/grains.ex:12]: Maybe the same thing here?

**Comment by neenjaw** [comment]
Also, with concept exercises (which are new) we have made a conscious choice to use the parentheses with typespecs since that is what you see in [official documentation](https://hexdocs.pm/elixir/typespecs.html).  They are also present in ~40% of the practice exercises (from what I saw doing a quick grep of the source files). So please add them.

We should probably go through and normalize them across all of the exercises.

**Review by neenjaw — APPROVED** [review]

**Comment by neenjaw** [comment]
Nice work! Thanks for this patch, really appreciate your help to make the track a better place!! 💙

**Comment by neenjaw** [comment]
Once v3 is released you'll get credit for this PR on the website too!

**Pull request merged**